test/benchmark/llvm_benchmark.cpp

```diff
@@ -126,10 +132,19 @@ void LLVMBenchmark::run_benchmark(const std::shared_ptr<ast::Program>& node) {
             // Log the time taken for each run.
             logger->info("Experiment {} compute time = {:.6f} sec", i, diff.count());
 
+            // Update statistics.
             time_sum += diff.count();
+            time_squared_sum += diff.count() * diff.count();
+            time_min = std::min(time_min, diff.count());
+            time_max = std::max(time_max, diff.count());
         }
         // Log the average time taken for the kernel.
-        logger->info("Average compute time = {:.6f} \n", time_sum / num_experiments);
+        double time_mean = time_sum / num_experiments;
+        logger->info("Average compute time = {:.6f}", time_mean);
+        logger->info("Compute time variance = {:.6f}",
```

Comment:
With some testing I did for small/large datasets this was always zero. Maybe it would be more meaningful if it was printed as `{:g}` for example?

Reply:
I fully agree, thanks for suggestion!